src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), universal selectors should not be used for setting global styles like margin, padding, etc. Consider creating a separate class for reusable styles or apply them to specific elements.